charts/pravega/README.md

```diff
@@ -158,7 +158,8 @@ The following table lists the configurable parameters of the pravega chart and t
 | `segmentStore.headlessSvcNameSuffix` | suffix for segmentStore headless service name | `pravega-segmentstore-headless` |
 | `segmentStore.labels` | Labels to add to the segmentStore pods | `{}` |
 | `segmentStore.annotations` | Annotations to add to the segmentStore pods | `{}` |
-| `segmentStore.initContainers` | Init Conatiners to add to the segmentStore pods | `[]` |
+| `segmentStore.initContainers` | Init Containers to add to the segmentStore pods | `[]` |
+| `authImplementations` | Plugin configuration to be used | `[]` |
```

Comment:
I think the README could use a tiny bit more guidance for:
- the user interested in the the authImplementation (the sample values.yaml is very helpful, but mentioning here that the user needs to provide an image with that implementing the AuthHandler (https://github.com/pravega/pravega/blob/master/documentation/src/docs/auth/auth-plugin.md); and the image should have /bin/sh available (sounds obvious, but it's not guaranteed, depending how the user crafts his/her Dockerfile.)
- for the generic controller segment store initcontainers, mention that it can be used in conjunction of the emptyDirVolume option you were mentioning elsewhere.

Calling all this out up front in the README will be very helpful.

Reply:
Updated the documentation